### ISSUE TITLE
Maybe improve the graphic resolution for the splash screen?

### ISSUE THREAD

**Issue by Raymo111** [leonzalion/TimeTract#3]
Kinda blurry atm

**Comment by leonzalion** [comment]
Yep, that's my fault. I acidentally made the original image too small and had to resize it. I am planning to redesign the logo soon, so keep your eyes peeled for that ;)
